cspreports/tests/test_models.py

```diff
@@ -168,14 +160,13 @@ def test_invalid_line_number(self):
         message = json.dumps(data)
         report = CSPReport.from_message(message)
 
-        self.assertTrue(report.is_valid)
         self.assertEqual(report.json, message)
         self.assertEqual(report.document_uri, 'http://protected.example.cz/')
         self.assertEqual(report.referrer, 'http://referrer.example.cz/')
         self.assertEqual(report.blocked_uri, 'http://dangerous.example.cz/')
         self.assertEqual(report.violated_directive, 'Very protective directive.')
         self.assertEqual(report.original_policy, 'Nothing is allowed.')
-        self.assertIsNone(report.line_number)
+        self.assertEqual(report.line_number, -666)
```

Comment:
This doesn't seem correct. The value may prevent report from being stored in database.

Reply:
Indeed, optional field should not cause the report to be invalid.